kytos/core/controller.py

```diff
@@ -605,6 +605,7 @@ async def event_handler(self, buffer_name: str):
         self.log.info(f"Event handler {buffer_name} started")
         while True:
             try:
+                await asyncio.sleep(0)
```

Comment:
@Alopalao let's also add a comment above this line explaining the rationale for this, just so any developer reading this in the future will understand why we added this, to avoid hogging too much capacity from the event loop on `MainThread` since this task can run very frequently. 

Also, changelog needs to be updated about this fix, just a general comment about potential starvation that has been fixed.

Reply:
Added comment.